src/main/java/com/backend/soullive_a/controller/ModelFitnessController.java

```diff
@@ -0,0 +1,36 @@
+package com.backend.soullive_a.controller;
+
+import com.backend.soullive_a.dto.request.ModelFitnessRequest;
+import com.backend.soullive_a.dto.response.model.fitness.ModelFitnessResponse;
+import com.backend.soullive_a.exception.base.BaseResponse;
+import com.backend.soullive_a.service.impl.ModelFitnessServiceImpl;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "모델 적합도 API", description = "모델 적합도 이슈 API입니다.")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/model/fitness")
+public class ModelFitnessController {
+    private final ModelFitnessServiceImpl modelFitnessService;
+
+    @Operation(summary = "모델 적합도 API", description = "모델 적합도 조회 API입니다.")
+    @GetMapping("")
+    public BaseResponse<ModelFitnessResponse> getModelFitness(
+            @RequestParam String modelName,
+            @RequestParam Long productId
+    ) {
+        return BaseResponse.<ModelFitnessResponse>builder()
+                .code(200)
+                .message("모델 적합도 조회에 성공했습니다.")
+                .isSuccess(true)
+                .data(modelFitnessService.getModelFitness(new ModelFitnessRequest(modelName, productId)))
```

Comment:
```suggestion
                .data(modelFitnessService.getModelFitness(String modelName, Long productId))
```
혹시 ModelFitnessRequest를 쓰는 이유가 있을까요..? 두개만 받는거면 그냥 인자로 넣어주면 service에서 getModelName 같은거 안해도 되니까 이렇게 하는게 코드가 좋을것같아요~
그리고 RequestDto를 쓰고 싶으면
`public BaseResponse<ModelFitnessResponse> getModelFitness(
            @RequestBody ModelFitness modelFitenessRequest
    )`
    이런식으로 쓰면 좋을것같아요

Reply:
오 requestbody에 dto가 되는줄 몰랏어요 대박 ,, ,,  지금 시간이 없어서 오늘 밤에 수정할게요 이거는